ctx.go

```diff
@@ -894,6 +900,21 @@ func (c *Ctx) JSON(data interface{}) error {
 	return nil
 }
 
+// ProblemJSON converts any interface or string to JSON.
+// Array and slice values encode as JSON arrays,
+// except that []byte encodes as a base64-encoded string,
+// and a nil slice encodes as the null JSON value.
+// This method also sets the content header to application/problem+json.
+func (c *Ctx) ProblemJSON(data interface{}) error {
```

Comment:
don't really know if we should make separate functions for all json possibilities, then love a functionality that expects the json and a custom json type

Reply:
Same as above, `ProblemJSON` changed to `CustomJSON`.